firmware/arduino/chessboard/chessboard.ino

```diff
@@ -36,6 +36,10 @@ int yMotor[4] = {MOTOR2_STEP_PIN, MOTOR2_DIR, MOTOR2_MS1, MOTOR2_MS2};
 #define Y_AXIS_ENDSTOP_SWITCH 32
 #define CHESS_TIMER_BUTTON 4
 
+// Distance Definitions
+#define MILLIMETERS_PER_SQUARE 200
```

Comment:
this should be 63, since that's around how many steps are in 2.5 inches. We'll end up making the board squares based on millimeters anyway so it'll line up nicely.